src/index.js

```diff
@@ -76,17 +79,43 @@ function validateFormData(data) {
 /*
  *    GENERATION
  */
-
 function generateTextInputAndLabel(formField) {
-  const textInputElement = document.createElement("input");
-  textInputElement.type = formField.type;
-  textInputElement.id = formField.id;
+  if (formField.multiline !== true) {
+    const textInputElement = document.createElement("input");
+    textInputElement.type = formField.type;
+    textInputElement.id = formField.id;
+
+    const textInputLabelElement = document.createElement("label");
+    textInputLabelElement.htmlFor = formField.id;
+    textInputLabelElement.innerText = formField.label;
+
+    return [textInputElement, textInputLabelElement];
+  } else {
+    const textInputElement = document.createElement("textarea");
+    textInputElement.id = formField.id;
+
+    const textInputLabelElement = document.createElement("label");
+    textInputLabelElement.htmlFor = formField.id;
+    textInputLabelElement.innerText = formField.label;
+
+    return [textInputElement, textInputLabelElement];
+  }
+}
+
+function generateRadioInputAndLabel(formField, x) {
```

Comment:
`x` here should be `optionData` (you're going to pass the optionData directly to the function)